docs/getting-started/net-map-req.asciidoc

```diff
@@ -14,7 +14,7 @@ configure `source.geo` and `destination.geo` ECS fields for your indices.
 [float]
 [[prereq-perms]]
 === Permissions required
-In order to view the map, you need at least `Read` privileges for `Maps`. To configure it, you need `All` privileges. Maps privilege settings are under *Kibana privileges* -> *Analytics* -> *Maps*.
+To view the map, you need a role with at least `Read` {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[privileges] for the `Maps` feature.
```

Comment:
That's how it was previously formatted, and historically we tend to use monospace for privileges (such as in the [Detections privs table](https://www.elastic.co/guide/en/security/master/detections-permissions-section.html#enable-detections-ui), "Kibana Privileges" column). I tend to agree that bold might be more appropriate for privs and features since they're UI elements, not text that the user types, but also hesitant to change this and make it less consistent with other feature privs.